ingredient/ingredient-container-reg/src/functions.ts

```diff
@@ -0,0 +1,23 @@
+import {BaseUtility, IngredientManager} from '@azbake/core'
+
+export class ContainerRegUtils extends BaseUtility {
+
+    public get_profile(): string {
+
+        let util = IngredientManager.getIngredientFunction("coreutils", this.context)
+        const acr_profile = util.create_resource_name("acr", null, false);
+        return acr_profile;
+    }
+    
+    public get_region(): string {
```

Comment:
Remove this, don't need to create a method that is a duplicate off coreutil

Reply:
Removed function.